install.sh

```diff
@@ -3545,10 +3552,16 @@ install_into_existing_repositories() {
         LOCAL_REPOSITORY_LIST=$(echo "$LOCAL_REPOSITORY_LIST" | sort)
     fi
 
-    for EXISTING in $LOCAL_REPOSITORY_LIST; do
+    # We dont want to change the std input through
+    # a while loop, so we redirect over descriptor 3
+    INPUT_PIPE=$(mktemp)
+    rm -rf "$INPUT_PIPE" >/dev/null 2>&1 || true
+    mkfifo "$INPUT_PIPE"
+    echo "$LOCAL_REPOSITORY_LIST" >"$INPUT_PIPE" &
+    while IFS= read -r EXISTING <&3; do
```

Comment:
Thats the best I could come up with which is POSIX...
If you have smoother way tell me, 
```shell
IFS="
"
for A in $A; do
   cmd-which-uses-std-in
done;
unset IFS
```
Does not work...